src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AppealSubmittedConfirmationTest.java

```diff
@@ -90,6 +95,40 @@ public void should_return_out_of_time_confirmation_when_out_of_time() {
         );
     }
 
+    @Test
+    public void should_return_confirmation_for_pay_offline_by_card_EA() {
```

Comment:
Try keep all to lowercase (standard format) for all unit test names

Reply:
renamed